src/main.ts

```diff
@@ -35,9 +46,16 @@ const parsedContext: GitHubContext = {
     inputs: context.payload.inputs,
     issue: issue ? { number: issue.number, node_id: issue.node_id as string } : undefined,
   },
+  config: { customField: projectFields },
 };
 
 synchronizer
   .synchronizeIssue(parsedContext)
-  .then(() => info("Finished"))
+  .then((result) => {
+    if (result) {
+      info("Operation finished successfully!");
+    } else {
+      error("There was a problem. Check logs for more information!");
+    }
+  })
   .catch(setFailed);
```

Comment:
What's the reason for having two different types of error conditions here (returning a `false`, and a `catch`)?

Reply:
It was from when I authorized the issue assignment to fail without crashing the application. I'll move it to simply fail.